treeherder/auth/backends.py

```diff
@@ -67,6 +67,19 @@ def _get_id_token_expiry(self, user_info):
         # https://openid.net/specs/openid-connect-core-1_0.html#IDToken
         return user_info['exp']
 
+    def _get_is_sheriff_from_userinfo(self, user_info):
+        """
+        Set users in sheriffing group in jwt response as is_staff
```

Comment:
not sure if this comment needs to be updated.

Reply:
This remains as is. `is_staff` refers to the cookie attribute and column name.